### ISSUE TITLE
Dyed clothes don't have their dyed colors in the right click menu

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#36495]
(WEB REPORT BY: angryturnip REMOTE: 172.93.101.77:7777)
# Revision
e8e4deae45e78a2e53bcff57ae1109be49343452

# Description
They just appear white
# Steps to Reproduce
Dye clothes, put them on, right click yourself
# What you Expected
Green jacket to be green
# What Actually Happened
https://files.catbox.moe/5hq3ac.png

**Comment by jwhitak** [comment]
![image](https://github.com/vgstation-coders/vgstation13/assets/69739118/36785b90-a103-4a1a-bce9-5752bbcb0a02)

**Comment by DeityLink** [comment]
Yeah, you don't have the correct hair color either. I'm not sure if the right-click menu is something built in byond, or if we can actually fix that.

**Comment by DeityLink** [comment]
Welp, it is, we're fucked.
